README.md

```diff
@@ -1,87 +1,141 @@
-> This simulator currently focuses on locomotion applications. Only contacts with ground are supported.
+Its design facilitates research and accelerates prototyping in the intersection of robotics and artificial intelligence. 
 
 ## Features
 
-- Physics engine in reduced coordinates implemented with [JAX][jax] in Python.
-- JIT compilation of Python code for increased performance.
-- Transparent support to execute logic on CPUs, GPUs, and TPUs.
-- Parallel multi-body simulations on hardware accelerators for significantly increased throughput.
-- Support for SDF models (and, upon conversion, URDF models).
-- Collision detection between bodies and uneven ground surface.
-- Soft contacts model supporting full friction cone and sticking / slipping transition.
-- Complete support for inertial properties of rigid bodies.
-- Revolute, prismatic, and fixed joints support.
-- Integrators: forward Euler, semi-implicit Euler, Runge-Kutta 4.
-- High-level classes for object-oriented programming.
-- High-level classes to compute multi-body dynamics quantities from the simulation state.
-- High-level classes wrapping the low-level functional RBDAs with support of [multiple velocities representations][notation].
-- Default validation of JAX pytrees to prevent JIT re-compilations.
-- Preliminary support for automatic differentiation of RBDAs.
+- Physics engine in reduced coordinates supporting fixed-base and floating-base robots.
+- Multibody dynamics library providing all the necessary components for developing model-based control algorithms.
+- Completely developed in Python with [`google/jax`][jax] following a functional programming paradigm.
+- Transparent support for running on CPUs, GPUs, and TPUs.
+- Full support for JIT compilation for increased performance.
+- Full support for automatic vectorization for massive parallelization of open-loop and closed-loop architectures.
+- Support for SDF models and, upon conversion with [sdformat][sdformat], URDF models.
+- Visualization based on the [passive viewer][passive_viewer_mujoco] of Mujoco.
+
+### JaxSim as a simulator:
+
+- Wide range of fixed-step explicit Runge-Kutta integrators.
+- Support for variable-step integrators implemented as embedded Runge-Kutta schemes.
+- All fixed-step and variable-step integrators support integrating the base orientation on the $\text{SO}(3)$ manifold.
+- Soft contacts model supporting full friction cone and sticking-slipping transition.
+- Collision detection between points rigidly attached to bodies and uneven ground surfaces.
+
+### JaxSim as a multibody dynamics library
+
+- Provides rigid body dynamics algorithms (RBDAs) like RNEA, ABA, CRBA, and Jacobians. 
+- Provides all the quantities included in the Euler-Poincarè formulation of the equations of motion.
+- Supports body-fixed, inertial-fixed, and mixed [velocity representations][notation].
+- Exposes all the necessary quantities to develop controllers in centroidal coordinates.
+
+### JaxSim for robot learning
```

Comment:
What about avoiding the repetition of `JaxSim`? Something like:
```suggestion
### For Advanced Simulation Capabilities:

- Wide range of fixed-step explicit Runge-Kutta integrators.
- Support for variable-step integrators implemented as embedded Runge-Kutta schemes.
- All fixed-step and variable-step integrators support integrating the base orientation on the $\text{SO}(3)$ manifold.
- Soft contacts model supporting full friction cone and sticking-slipping transition.
- Collision detection between points rigidly attached to bodies and uneven ground surfaces.

### For Multibody Dynamics Computations

- Provides rigid body dynamics algorithms (RBDAs) like RNEA, ABA, CRBA, and Jacobians. 
- Provides all the quantities included in the Euler-Poincarè formulation of the equations of motion.
- Supports body-fixed, inertial-fixed, and mixed [velocity representations][notation].
- Exposes all the necessary quantities to develop controllers in centroidal coordinates.

### For Robot Learning

Reply:
I don't like much starting with `For`, I probably prefer more the repetition to mark explicitly that JaxSim (despite the name) is not only a simulator.